### PR TITLE
NIFI-2750 Add Optional Column Name Quoting to ConvertJSONToSQL

### PR DESCRIPTION
**Pull request by patricker**
Adds quoted identifier support to ConvertJSONToSQL; enabled through an optional property, false by default.

Thank you for submitting a contribution to Apache NiFi.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [x] Is there a JIRA ticket associated with this PR? Is it referenced 
     in the commit message?

- [x] Does your PR title start with NIFI-XXXX where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.

- [x] Has your PR been rebased against the latest commit within the target branch (typically master)?

- [x] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you ensured that the full suite of tests is executed via mvn -Pcontrib-check clean install at the root nifi folder?
- [x] Have you written or updated unit tests to verify your changes?
- [x] If adding new Properties, have you added .displayName in addition to .name (programmatic access) for each of the new properties?

### CODE REVIEW

File: nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ConvertJSONToSQL.java [hunk 1]
```diff
@@ -254,6 +264,9 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
         final boolean failUnmappedColumns = FAIL_UNMATCHED_COLUMN.getValue().equalsIgnoreCase(context.getProperty(UNMATCHED_COLUMN_BEHAVIOR).getValue());
         final boolean warningUnmappedColumns = WARNING_UNMATCHED_COLUMN.getValue().equalsIgnoreCase(context.getProperty(UNMATCHED_COLUMN_BEHAVIOR).getValue());
 
+        //Escape column names?
+        final boolean escapeColumnNames = "true".equalsIgnoreCase(context.getProperty(QUOTED_IDENTIFIERS).getValue());
```
Comment (pvillard31) [inline nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ConvertJSONToSQL.java]: I'd go for:
````java
context.getProperty(QUOTED_IDENTIFIERS).asBoolean()
````

**Comment by patricker** [comment]
@trixpan Thanks for the feedback, didn't know about the `asBoolean` option.  Updated code.

**Comment by mattyb149** [comment]
Reviewing...

**Comment by mattyb149** [comment]
+1 LGTM, ran unit tests and tested with MySQL, verified the default behavior is preserved, and column names are quoted correctly when specified.  Thank you for the contribution! Merging to master

**Pull request closed without merging**